solidity/test/token_stake/TestTokenStake.js

```diff
@@ -147,9 +148,10 @@ contract('TokenStaking', function(accounts) {
   it("should allow to cancel delegation just before initialization period is over", async () => {
     let ownerStartBalance = await token.balanceOf.call(ownerOne);
     
-    await delegate(operatorOne, stakingAmount);
+    let tx = await delegate(operatorOne, stakingAmount)
+    let createdAt = (await web3.eth.getBlock(tx.receipt.blockNumber)).timestamp
 
-    await increaseTime(initializationPeriod);
+    await increaseTimeTo(createdAt + initializationPeriod - 1)
```

Comment:
Why we are `-1` here? [Per the previous comment](https://github.com/keep-network/keep-core/pull/1512/files/d339004f82283c04495c8638676b7e87b08fe6c2#r397880959):

> *just before initialization period is over* is at `increaseTime(initializationPeriod)`